content/en/docs/getting-started/prerequisites/hardware/snp.md

```diff
@@ -8,5 +8,107 @@ tags:
 - SEV
 ---
 
-TODO
+## Platform Setup
+
+In order to launch SEV or SNP memory encrypted guests, the host must be prepared with a compatible kernel, `6.8.0-rc5-next-20240221-snp-host-cc2568386`. AMD custom changes and required components and repositories will eventually be upstreamed. 
```

Comment:
This is inaccurate. The an upstream host kernel can be used for SEV. This page is label as SNP instructions anyway so maybe just remove the mention of SEV.